src/provisioner/provisioner.py

```diff
@@ -48,12 +50,25 @@ def config_apply(solution_conf_url: str, cortx_conf_url: str = None):
         """
         if Log.logger is None:
             CortxProvisionerLog.initialize(const.SERVICE_NAME, const.TMP_LOG_PATH)
-        Log.info('Applying config %s' % solution_conf_url)
-        Conf.load(CortxProvisioner._solution_index, solution_conf_url)
 
         if cortx_conf_url is None:
             cortx_conf_url = CortxProvisioner._cortx_conf_url
         cortx_config_store = ConfigStore(cortx_conf_url)
+
+        # Check if config is already applied.
+        if cortx_config_store.get('cortx>common>release') and not conf_override:
```

Comment:
Other patch has cortx_config_store renamed to cortx_conf. So while merging you will have to take care.

Reply:
Sure.